.github/workflows/ci.yml

```diff
@@ -360,7 +360,7 @@ jobs:
         run: cargo doc --lib --no-deps --all-features --document-private-items
         env:
           RUSTFLAGS: --cfg docsrs
-          RUSTDOCFLAGS: --cfg docsrs -Dwarnings
+          RUSTDOCFLAGS: --cfg docsrs --cfg unstable_tokio -Dwarnings
```

Comment:
is this backwards?

Reply:
🤦 Yes. I swear I thought I double-checked that.  I also see I missed it on the RUSTFLAGS line above.  Fixing...  Thanks @LucioFranco